en/docs/references/test-your-service-via-postman.md

```diff
@@ -0,0 +1,145 @@
+# Test Your Service via Postman
+
+The Postman Editor allows you to integrate with your Postman account to easily test your services.
```

Comment:
I feel that users. will not understand what "Postman Editor" is here.
Shall we improve this description to give the idea that Choreo allows users to test their services using Postman.

Reply:
Updated.